__init__.py

```diff
@@ -2,11 +2,12 @@
 from __future__ import unicode_literals
 import json, os
 import six
+from io import open
 
 try:
     langs
 except:
-    with open(os.path.join(os.path.dirname(__file__), 'langs.json')) as f:
+    with open(os.path.join(os.path.dirname(__file__), 'langs.json'), encoding='utf-8') as f:
         langs = json.load(f)
```

Comment:
This code is good. It works on both Python 2 and 3...

However, while I was testing it I found out that there is no straightforward way to dump json to a file that's compatible with both Python 2 and 3. Here's an example of how it can be done, FYI:
```py
from __future__ import unicode_literals
from __future__ import print_function
import os
import json
from io import open

import six


path = "/tmp/data.json"
data = {"key": "\u4500"}
print(data)

'''
# This should work, but it doesn't: works on Python 3, not Python 2.

with open(path, "w", encoding="utf-8") as wf:
    json.dump(data, wf)
'''

with open(path, "w", encoding="utf-8") as wf:
    json_data = json.dumps(data)
    if six.PY2:
        # ugly hack because of bugs in Python 2 json.dump(s)
        assert isinstance(json_data, bytes), type(json_data)
        json_data = json_data.decode('utf-8')
    wf.write(json_data)

# read from file: works on both Python 2 and 3
with open(path, encoding="utf-8") as rf:
    new_data = json.load(rf)

print(new_data)
```

Reply:
That's good to know - thanks for informing.